File: beecrowd 1036 - Fórmula de Bhaskara.cpp
#include <iostream>
#include <cmath>
using namespace std;
int main() {
  double a,b,c,x,z, delta;
  cin >> a >> b >> c;
  delta = pow(b,2)-4*a*c;
  x = (-b + pow(delta,0.5))/(2*a);
  z = (-b - pow(delta,0.5))/(2*a);

  if (a == 0 || delta < 0){
    cout << "Impossivel calcular\n";   
  }
  else {
    cout.precision(5);
    cout << "R1 = " << fixed << x << endl << "R2 = " << z << endl;
  }
  return 0;
}

File: beecrowd 1036 - Fórmula de Bhaskara.py
a,b,c = map(float,input().split())
delta = (b**2)-4*a*c
if delta < 0 or a == 0:
  print('Impossivel calcular')
else:
  x1 = (-b+(delta**0.5))/(2*a)
  x2 = (-b-(delta**0.5))/(2*a)
  print("R1 = %.5f\nR2 = %.5f"%(x1,x2))
